scripts/install_dependencies/install_openvino_dependencies.sh

```diff
@@ -200,7 +200,7 @@ elif [ "$os" == "centos7" ] ; then
     # find -name *.so -exec objdump -p {} \; | grep NEEDED | sort -u | cut -c 23- | xargs -t -n1 yum -q whatprovides
 
     pkgs_opencv_req=(gtk2)
-    pkgs_python=(python3 python3-devel python3-setuptools python3-pip)
+    pkgs_python=(python3 python3-devel python3-setuptools python3-pip python36-gi python36-gobject python36-gobject-devel)
```

Comment:
if it needs only for dl streamer, please move these packages to dl_streamer component. Python component is common for all docker images.